src/Symfony/Component/Translation/Dumper/YamlFileDumper.php

```diff
@@ -36,4 +36,68 @@ protected function getExtension()
     {
         return 'yml';
     }
+
+    protected function expandToTree(array $messages)
```

Comment:
methods should be protected (there is no reason to make them extension points yet)